src/main/java/edu/kis/powp/jobs2d/command/ComprisingCommandsCounterVisitor.java

```diff
@@ -0,0 +1,54 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ComprisingCommandsCounterVisitor implements CommandVisitorInterface
+{
+    private Job2dDriver job2dDriver;
+    private int operateToCounter = 0;
+    private int setPositionCounter = 0;
+    private int CompoundCounter = 0;
+
+    public int getOperateToCounter()
+    {
+        return operateToCounter;
+    }
+
+    public int getSetPositionCounter()
+    {
+        return setPositionCounter;
+    }
+
+    public int getCompoundCounter()
+    {
+        return CompoundCounter;
+    }
+
+    public ComprisingCommandsCounterVisitor(Job2dDriver driver)
+    {
+        this.job2dDriver = driver;
+    }
+
+    @Override
+    public void visit(OperateToCommand driver)
+    {
+        operateToCounter++;
+    }
+
+    @Override
+    public void visit(SetPositionCommand driver)
+    {
+        setPositionCounter++;
+    }
+
+    @Override
+    public void visit(ICompoundCommand driver)
+    {
+        driver.iterator().forEachRemaining(c -> c.accept(this));
```

Comment:
OK! Why you use forEachRemaining instead of forEach? Are there any particular "remains"?